boards/arm/stm32_min_dev/Makefile.board

```diff
@@ -0,0 +1,8 @@
+FLASH_SCRIPT = openocd.sh
+DEBUG_SCRIPT = openocd.sh
+
+FLASH_BASE_ADDRESS = 0x08000000
```

Comment:
FLASH_BASE_ADDRESS should come from the devicetree/kconfig, we are working on a fix for this, so this should be replaced with CONFIG_FLASH_BASE_ADDRESS